source/simulator/core.cc

```diff
@@ -1746,6 +1746,8 @@ namespace aspect
       constraints.distribute (old_distributed_system);
       old_solution = old_distributed_system;
 
+      current_linearization_point = distributed_system;
```

Comment:
Can you leave a comment here that summarizes the discussion (that we need some value for the inflow/outflow determination and that it will be updated later)